src/genie/libs/parser/iosxr/show_bgp.py

```diff
@@ -5434,7 +5436,14 @@ def cli(self, output=None):
                 vrf = group['vrf']
                 neighbor = group['neighbor']
                 spk = int(group['spk'])
-                as_number = int(group['as_number'])
+
+                as_number = "NA"
+                try:
+                    as_number = int(group['as_number'])
+                except:
```

Comment:
```suggestion
                except Exception:
```
This is needed to avoid issues with ctrl+c